### PR TITLE
added The Economist to sources

### PR DESCRIPTION
**Pull request by susi189**
Liked to Issue #49 

- Adding new news source: The Economist.

- Prettier formatting: added spaces to the region key value pair

### CODE REVIEW

**Comment by susi189** [comment]
Hi @MizouziE this is my first pull request here :) So hopefully done everything right, let me know if there are any issues.

File: src/data/sources.json [hunk 1]
```diff
@@ -3,102 +3,108 @@
...
     "site": "https://www.euronews.com/",
     "base": "https://www.euronews.com",
-    "region":"Europe"
+    "region": "Europe"
+  },
+  {
+    "name": "The Economist",
+    "site": "https://www.economist.com/",
+    "base": "https://www.economist.com/",
+    "region": "Europe"
```
Comment (MizouziE) [inline src/data/sources.json:108]: Nice addition, thank you!

**Review by MizouziE — APPROVED** [review]
Appreciate the formatting also

**Comment by MizouziE** [comment]
@all-contributors please add @susi189 for code

**Pull request merged**